portfolio/src/constants.ts

```diff
@@ -0,0 +1,133 @@
+import AmazonNavigatorImgSrc from "./assets/Project1.png";
+import PlantParenthoodImgSrc from "./assets/Project2.png";
+import PasswordGeneratorImgSrc from "./assets/Project3.png";
+import TextEditorImgSrc from "./assets/Project4.png";
+import { ExperienceLevel } from "./types";
+
```

Comment:
I went ahead and moved all of the constants to a different file. This just helps to clean up the components themselves. Anytime you can simply things or organize things, the better. It will make your code a lot more readable and make it easier for you to find definitions later on given that you won't have to go searching within your component files for these. In addition, extracting these definitions to this file makes it so that you could, if you wanted or needed to, share them with other components.

Reply:
Gotcha, sometimes I have trouble with not knowing where the individual code belong, but it seems to me it's similar to child-parent relationship, small amounts of constant code can be placed in the same "component" so I wouldn't have to waste time updating each line of code